pkgs/servers/gerbera/default.nix

```diff
@@ -0,0 +1,77 @@
+{ stdenv, fetchgit
+, cmake, pkg-config
+# required
+, libupnp, libuuid, pugixml, libiconv, sqlite, zlib, spdlog, fmt
+, pkgs
+# options
+, enableDuktape ? true
+, enableCurl ? true
+, enableTaglib ? true
+, enableLibmagic ? true
+, enableLibmatroska ? true
+, enableAvcodec ? false
+, enableLibexif ? true
+, enableExiv2 ? false
+, enableFFmpegThumbnailer ? false
+, enableInotifyTools ? true
+}:
+
+with stdenv.lib;
+let
+  optionOnOff = option: if option then "on" else "off";
+in stdenv.mkDerivation rec {
+  pname = "gerbera";
+  version = "1.5.0";
+
+  src = fetchgit {
+    url = meta.repositories.git;
+    rev = "0c486fc53b097227db2443c738647690635bdf99";
```

Comment:
Why not `rev = "v${version}"`?

Reply:
That's a good question...
I don't recall my reasoning there, i'll adapt.